### DISCUSSION TITLE
Enhance and add missing Arabic (AR) Translation from English

### DISCUSSION THREAD

**Discussion by EG-Mohamed** [Laravel-Backpack/community-forum#509] [General]
@tabacitu @pxpm 
Hello guys great work on v6 and I'd like to be a little part of it so I want to re-translate it from English to Arabic (my native language) so it will be ready to go with multiple languages including Arabic and maybe add a language switch in the menu bar as well.

Done ✌💪
https://github.com/Laravel-Backpack/CRUD/pull/5088

**Comment by tabacitu** [comment]
Awesome, thanks a lot @EGYWEB-Mohamed ! Feel free to translate other repos you use, happy to merge!

**Reply by EG-Mohamed** [reply]
Sure i will do it asap :)
